### ISSUE TITLE
Can't establish connection when there is no user 'pi' on server

### ISSUE THREAD

**Issue by MaxLoep** [cyclotron-bonn/irrad_control#44]
The irrad_control software can't establish a ssh connection to the raspberry server if there is no user with the name 'pi'.

**Comment by leloup314** [comment]
That's true. Currently, the "design choice" (its not really a bug in this sense) is that the server user  is always ``pi`` .
For better flexibility and because its bad practice to hard-code usernames this should be changed!
